libs/feature/map/src/lib/map-context/map-context.service.ts

```diff
@@ -30,6 +31,11 @@ export const DEFAULT_BASELAYER_CONTEXT: MapContextLayerXyzModel = {
   ],
 }
 
+export const DEFAULT_VIEW: MapContextViewModel = {
+  center: fromLonLat([2.1, 46.8], 'EPSG:3857'),
```

Comment:
maybe it would be a good time to not have metropolitan France as a default extent any more :) a global world view would make more sense IMO

Reply:
true, I'll adapt this, only drawback will be that datasets in France for which the viewer falls back to this default, will become less "visible"